### ISSUE TITLE
Textfield onBlur / onFocus handlers do not repond in Firefox

### ISSUE THREAD

**Issue by debois** [debois/elm-mdl#194]
This is a regression caused by the fix to #166. We'll correct it once #179 is merged. 

However, it doesn't seem we can fix this without bumping the major version, so this is unlikely to be fixed in v7.

**Comment by vipentti** [comment]
Temporary fix to enable these is to include a polyfill. 

One could use polyfill.io for example
```html
<script src="https://cdn.polyfill.io/v2/polyfill.min.js"></script>
```

or  https://github.com/paulcpederson/focusin

**Comment by debois** [comment]
Oh, cool! Can you update the demo to check if we're on Firefox and include the polyfill; and mention it in the docs for onBlur / onFocus? (No need for a PR.)

**Comment by vipentti** [comment]
This was fixed (for now) in e533afb
